### PR TITLE
Added the Python plugin for the Tyk middleware to check the Approov token in the pre hook.

### PR DESCRIPTION
**Pull request by Exadra37**
The Python plugin allows to verify the Approov token without interfering with user authentication or API keys checks.

Signed-off-by: Exadra37 <exadra37@gmail.com>

### CODE REVIEW

File: docs/APPROOV_TOKEN_PYTHON_PLUGIN_QUICKSTART.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+Adding the API domain also configures the [dynamic certificate pinning](https://approov.io/docs/latest/approov-usage-documentation/#approov-dynamic-pinning) setup, out of the box.
+
+> **NOTE:** By default the pin is extracted from the public key of the leaf certificate served by the domain, as visible to the box issuing the Approov CLI command and the Approov servers.
+
+### Approov Secret
+
+Approov tokens are signed with a symmetric secret. To verify tokens, we need to grab the secret using the [Approov secret command](https://approov.io/docs/latest/approov-cli-tool-reference/#secret-command) and plug it into the Tyk API Gateway configuration to check the signatures of the [Approov Tokens](https://www.approov.io/docs/latest/approov-usage-documentation/#approov-tokens) that it processes.
+
+
+To retrieved the Approov secret you need to enable your `admin` role with:
```
Comment (richardmtaylor) [inline docs/APPROOV_TOKEN_PYTHON_PLUGIN_QUICKSTART.md:64]: Typo "retrieved" should be "retrieve"

File: docs/APPROOV_TOKEN_PYTHON_PLUGIN_QUICKSTART.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+```bash
+eval `approov role admin`
```
Comment (richardmtaylor) [inline docs/APPROOV_TOKEN_PYTHON_PLUGIN_QUICKSTART.md:67]: I thought you'd added a note in other quickstarts that a different syntax is needed on Windows?

File: docs/APPROOV_TOKEN_PYTHON_PLUGIN_QUICKSTART.md (hunk 1 above)
Comment (Exadra37) [reply docs/APPROOV_TOKEN_PYTHON_PLUGIN_QUICKSTART.md:64]: Good catch. Will fix it.

File: docs/APPROOV_TOKEN_PYTHON_PLUGIN_QUICKSTART.md (hunk 2 above)
Comment (Exadra37) [reply docs/APPROOV_TOKEN_PYTHON_PLUGIN_QUICKSTART.md:67]: Only for the Azure quickstart. I will add it to all.

**Pull request merged**